main.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import json
+
+
+def create_json(json_gps, latitude, longitude):
+    x = dict(latitude = latitude, longitude = longitude)
+    json_gps.append(x)
```

Comment:
You don't need to add a json_gps, just print it (emulate request-response: after each request you will need to response with only 1 set of gps coordinates, not a list)